models/packages/package.go

```diff
@@ -130,6 +130,8 @@ type Package struct {
 	Name             string `xorm:"NOT NULL"`
 	LowerName        string `xorm:"UNIQUE(s) INDEX NOT NULL"`
 	SemverCompatible bool   `xorm:"NOT NULL DEFAULT false"`
+	Description      string `xorm:"VARCHAR(255)"`
+	Readme           string `xorm:"LONGBLOB"`
```

Comment:
I believe TEXT is more suitable for this column, as xorm should translate it into the appropriate type for each database

Reply:
`LONGTEXT` (for MySQL)